raiden/network/proxies/token_network_registry.py

```diff
@@ -104,18 +136,20 @@ def add_token(self, token_address: TokenAddress, given_block_identifier: BlockSp
 
         checking_block = self.client.get_checking_block()
         error_prefix = 'Call to createERC20TokenNetwork will fail'
+
+        arguments = [token_address] + additional_arguments
         gas_limit = self.proxy.estimate_gas(
             checking_block,
             'createERC20TokenNetwork',
```

Comment:
I thought we would merge the two smart contracts into one, which would allows us to remove the limits and have an easy upgrade, did that change?

Reply:
Related issue https://github.com/raiden-network/raiden-contracts/issues/694

Reply:
We always have limits.  Sometimes it's `MAX_UINT256`, meaning no limits.